src/test/java/apicurito/tests/steps/verification/PathVerifications.java

```diff
@@ -45,25 +47,38 @@ public void checkThatPathParameterIsCreatedForPath(String parameter, String path
         }
 
         SelenideElement parameterElement = PathUtils.getPathPageRoot().$(PathElements.PATH_PARAMETERS_SECTION).$$(PathElements.PATH_PARAMETERS_ROW).filter(matchText(parameter)).first();
-        CollectorHelper.getCollector().assertThat(parameterElement.$("div").getAttribute("class")).as("Path parameter %s is not created", parameter).doesNotContain("missing");
+        assertThat(parameterElement.$("div").getAttribute("class")).as("Path parameter %s is not created", parameter).doesNotContain("missing");
     }
 
     @Then("^check that path parameter \"([^\"]*)\" has description \"([^\"]*)\"$")
-    public void checkThatPathParameterHasDescription(String parameter, String description) {
+    public void checkThatPathParameterHasDescription(String parameter, String description) {        //NEW
         SelenideElement descriptionElement = PathUtils.getPathPageRoot().$(PathElements.PATH_PARAMETERS_SECTION).$$(PathElements.PATH_PARAMETERS_ROW)
                 .filter(matchText(parameter)).first().$(By.className("description"));
 
-        CollectorHelper.getCollector().assertThat(descriptionElement.getText()).as("Description for path parameter %s is different", parameter).isEqualTo(description);
+        assertThat(descriptionElement.getText()).as("Description for path parameter %s is different", parameter).isEqualTo(description);
     }
 
-    @Then("^check that path parameter \"([^\"]*)\" has type \"([^\"]*)\" formatted as \"([^\"]*)\"$")
-    public void checkThatPathParameterHasTypeFormattedAs(String parameter, String type, String as) {
-        SelenideElement summaryElement = PathUtils.getPathPageRoot().$(PathElements.PATH_PARAMETERS_SECTION).$$(PathElements.PATH_PARAMETERS_ROW)
-                .filter(matchText(parameter)).first().$(By.className("summary"));
+    @Then("^check that path parameter \"([^\"]*)\" has type \"([^\"]*)\"$")
+    public void checkThatPathParameterHasType(String parameter, String expectedType) {       //NEW
+        PathUtils.openPathTypes(parameter);
+        String type = PathUtils.getPathPageRoot().$(PathElements.PATH_PARAMETERS_SECTION).$$(PathElements.PATH_PARAMETERS_ROW)
```

Comment:
We should add these two rows into separate method with one parameter - the element we are looking for. The usage would be something like: `getTextOfParameter(PROPERTY_TYPE)`, wdyt?

Reply:
I agree. I will do it later. I am creating jira for that